Dockerfile

```
@@ -14,7 +14,7 @@ RUN go mod download
 COPY . .
 RUN go build -a -o gemini
 
-FROM alpine:3.15
+FROM alpine:3.16.2
```

We like to unpin the patch version
```suggestion
FROM alpine:3.17
```